### PR TITLE
11 add speaker diarization

### LINKED ISSUE #11: Add speaker diarization

### PR DESCRIPTION
**Pull request by ibevers**

### CODE REVIEW

File: src/senselab/audio/tasks/pyannote_speaker_diarization_31.py
Comment (fabiocat93) [inline src/senselab/audio/tasks/pyannote_speaker_diarization_31.py:1]: hey @ibevers ! Can you please provide the model name and revision parameters of the methods? In this way, one may call v2.0, v3.0, or v3.1 with the same code, and your script would be more generalizable.

File: src/senselab/cli.py
Comment (fabiocat93) [inline src/senselab/cli.py:1]: I think this requires one more " at the beginning of the file

**Comment by fabiocat93** [comment]
hi @ibevers , thank you for your work! please make sure all tests pass

**Comment by ibevers** [comment]
> hi @ibevers , thank you for your work! please make sure all tests pass

Will fix this today!

**Comment by ibevers** [comment]
Fixed! @fabiocat93

File: src/senselab/audio/tasks/pyannote_speaker_diarization_31.py
Comment (ibevers) [reply src/senselab/audio/tasks/pyannote_speaker_diarization_31.py:1]: Ah, yes, will do

File: src/senselab/cli.py
Comment (ibevers) [reply src/senselab/cli.py:1]: Fixed

File: src/senselab/audio/tasks/pyannote_speaker_diarization_31.py
Comment (ibevers) [reply src/senselab/audio/tasks/pyannote_speaker_diarization_31.py:1]: done

**Comment by ibevers** [comment]
@fabiocat93 would you mind reviewing this again and merging if everything looks okay?

**Comment by fabiocat93** [comment]
> @fabiocat93 would you mind reviewing this again and merging if everything looks okay?

hi @ibevers , you should first update it as in the `main` branch (this is 14 commits behind)

**Comment by ibevers** [comment]
Done @fabiocat93

**Review by fabiocat93 — APPROVED** [review]

**Pull request merged**